feat: Finished csv import scripts [PT-188117054]

dougmartin opened the pull request.

dougmartin committed:
feat: Finished csv import scripts [PT-188117054]

pjanik reviewed (changes requested):
It looks great. 👍 

The only thing we need to change is the CopyPlugin. I don't think we can use it these days, as it doesn't work with the new release script and generally it's not the best approach compared to assets loader (caching, etc.).

We should replace it with an assets loader that will generate a URL. I'm pretty sure the default limit for inlining is smaller than the size of this file, but we can also explicitly set this path to use the `asset/resource` type.

We can also merge this PR, and I can do it myself later - whichever you prefer.

dougmartin committed:
Removed copy-webpack-plugin

The copy-webpack-plugin was added in the initial commit to copy the values.json from the public folder.  Instead the values.json file is imported into the code but webpack is configured to return the filename instead of the data for the file so that we can call fetch with the filename.

pjanik reviewed (approved):
Updates look good, thanks.
